src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/hosts.po.ts

```diff
@@ -49,7 +49,6 @@ export class HostsPageHelper extends PageHelper {
       }
       if (exist) {
         cy.get('#hostname').should('have.class', 'ng-invalid');
-        cy.get('cd-back-button').click();
```

Comment:
Makes sense, but what happened to this button? It'd be great if you add some more context to the PR description:
```
 1) Hosts page
      when Orchestrator is available
        should not add an exsiting host:
    AssertionError: Timed out retrying after 120000ms: Expected to find element: `cd-submit-button`, but never found it.
```